pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1150,6 +1150,7 @@ func TestSingleConstraint(t *testing.T) {
 		nodes        []*v1.Node
 		existingPods []*v1.Pod
 		fits         map[string]bool
+		wantErrCode  map[string]int
```

Comment:
please do `map[string]status.Code` and use constants instead of numbers.

Reply:
Done, pls help to check if that is what you want.